pep-0630.rst

```diff
@@ -321,10 +328,26 @@ the Python level: for example, you can't set ``str.myattribute = 123``.
    that shares any Python objects across interpreters implicitly depends
    on CPython's current, process-wide GIL.
 
-An alternative to static types is *heap-allocated types*, or heap types
+Because they are immutable and process-global, static types cannot access
+“their” module state.
+If any method of such a type requires access to module state,
+the type must be converted to a *heap-allocated type*, or *heap type*
 for short. These correspond more closely to classes created by Python’s
 ``class`` statement.
 
+For new modules, using heap types by default is a good rule of thumb.
+
+Existing classes can be converted to heap ones, but note that
+the heap type API was not designed for “lossless” conversion
```

Comment:
"Lossless" is a little vague IMO. Could you reword this with a clearer language?